tests/Functional/BundleConfigurationTest.php

```diff
@@ -22,12 +32,17 @@ public function provideBundleWithDifferentConfigurationFormats()
             [__DIR__.'/../Fixtures/Resources/ConfigurationBundle/config.yml'],
             [__DIR__.'/../Fixtures/Resources/ConfigurationBundle/config.xml'],
             [__DIR__.'/../Fixtures/Resources/ConfigurationBundle/config.php'],
-            [function (ContainerBuilder $container) {
-                $container->loadFromExtension('configuration', [
-                    'foo' => 'val1',
-                    'bar' => ['val2', 'val3'],
-                ]);
-            }],
+            [
+                function (ContainerBuilder $container) {
+                    $container->loadFromExtension(
+                        'configuration',
+                        [
+                            'foo' => 'val1',
+                            'bar' => ['val2', 'val3'],
+                        ]
+                    );
+                },
+            ],
```

Comment:
Is this change just CS?

I would prefer that PRs does not include CS changes on unrelated code.

Reply:
Was unintended, I will revert this here.